scripts/evaluate_pushing_policy.py

```diff
@@ -0,0 +1,135 @@
+    # from pathlib import Path
+
+    # from gymnasium.wrappers import RecordVideo
+
+    # if render:
+    #     video_folder = "videos/blocks2d-rl-improvisational-evaluation"
+    #     Path(video_folder).mkdir(parents=True, exist_ok=True)
+    #     env = RecordVideo(
+    #         env,
+    #         video_folder,
+    #         episode_trigger=lambda _: True,  # Record all episodes
+    #         name_prefix="evaluation",
+    #     )
+
+    # Load policy
+    policy = RLImprovisationalPolicy(env)
+    policy.load(policy_path)
+
+    success_count = 0
+    episode_lengths = []
+    rewards_history = []
+
+    # Run episodes
+    for episode in range(num_episodes):
+        # obs, _ = env.reset(seed=seed + episode if seed is not None else None)
+        obs, _ = env.reset(seed=seed if seed is not None else None)
+
+        if debug:
+            print(f"\nEpisode {episode + 1}:")
+            print(f"Initial robot position: ({obs[0]:.3f}, {obs[1]:.3f})")
+            print(f"Initial block 2 position: ({obs[6]:.3f}, {obs[7]:.3f})")
+            print(f"Target area: ({obs[11]:.3f}, {obs[12]:.3f})")
+
+        episode_length = 0.0
+        episode_reward = 0.0
+
+        while True:
+            # Get action from policy
+            action = policy.get_action(obs)
+
+            if debug:
+                print(f"\nStep {episode_length + 1}:")
+                print(f"Action: {action}")
+
+            obs, reward, terminated, truncated, _ = env.step(action)
+
+            if debug:
+                print(f"New robot position: ({obs[0]:.3f}, {obs[1]:.3f})")
+                print(f"New block 2 position: ({obs[6]:.3f}, {obs[7]:.3f})")
+                print(f"Reward: {reward:.3f}")
+
+            episode_length += 1
+            episode_reward += reward
+
+            if terminated or truncated:
+                if terminated:  # Successfully cleared area
+                    success_count += 1
+                episode_lengths.append(episode_length)
+                rewards_history.append(episode_reward)
+
+                print(
+                    f"Episode {episode + 1}: "
+                    f"{'Succeed' if terminated else 'Fail'} in {episode_length} steps."
+                    f"Total reward: {episode_reward:.1f}"
+                )
+                break
+
+    # Calculate and return results
+    results = {
+        "success_rate": success_count / num_episodes,
+        "avg_episode_length": np.mean(episode_lengths),
+        "avg_reward": np.mean(rewards_history),
+        "min_length": min(episode_lengths),
+        "max_length": max(episode_lengths),
+        "successes": success_count,
+        "total_episodes": num_episodes,
+    }
+
+    print("\nEvaluation Summary:")
+    print(f"Success Rate: {results['success_rate']:.1%}")
+    print(f"Average Episode Length: {results['avg_episode_length']:.1f}")
+    print(f"Success Count: {success_count}/{num_episodes}")
+
+    return results
+
+
+def test_evaluate_pushing_policy():
```

Comment:
are you currently running this file with pytest? since this is just a script and not a unit test, can we change the structure and run it with python instead? for example you could add something like
```
if __name__ == "__main__":
    evaluate_pushing_policy(...)
```